jbt-it/neomdb
# Added "loading circle" to app

Added special loading circle component.

How to test:
- Navigate to a page
- Reload the page to see briefly the loading circle
